README.md

```diff
@@ -1,8 +1,32 @@
 [![Build Status](https://travis-ci.org/alex-konovalov/gap-lesson.svg?branch=gh-pages)](https://travis-ci.org/alex-konovalov/gap-lesson)
 [![DOI](https://zenodo.org/badge/44778145.svg)](https://zenodo.org/badge/latestdoi/44778145)
 
-This is the lesson on the computational algebra system [GAP](http://www.gap-system.org).
+This is the Carpentries-style lesson on the computational algebra system [GAP](http://www.gap-system.org).
 
-It is currently being developed by Alexander Konovalov for the Software Carpentry
-workshop which will be a part of the [First CoDiMa Training School in Computational
-Discrete Mathematics](http://www.codima.ac.uk/school2015/).
+Its development has been started by Alexander Konovalov for the Software Carpentry
+workshop organised as a part of the [First CoDiMa Training School in Computational
+Discrete Mathematics](http://www.codima.ac.uk/school2015/) in 2015. Since then, it
+had several releases, produced in collaboration with further contributors, and had
```

Comment:
"**has** had several releases", since we're still making new ones.

"and **has** been taught", since it'll be taught again! :)